src/components/dialogs/parameters/loadflow/load-flow-parameters.tsx

```diff
@@ -0,0 +1,470 @@
+                                    )}
+                                </Grid>
+                            </Grid>
+                        </Box>
+                        <Box sx={{ flexGrow: 0 }}>
+                            <LineSeparator />
+                            <Grid
+                                container
+                                item
+                                sx={mergeSx(styles.controlParametersItem, styles.marginTopButton, { paddingBottom: 0 })}
+                            >
+                                <LabelledButton
+                                    callback={() => setOpenSelectParameterDialog(true)}
+                                    label="settings.button.chooseSettings"
+                                />
+                                <LabelledButton callback={() => setOpenCreateParameterDialog(true)} label="save" />
+                                <LabelledButton callback={resetLFParametersAndProvider} label="resetToDefault" />
+                                <LabelledButton label="resetProviderValuesToDefault" callback={resetLFParameters} />
+                                <SubmitButton
+                                    onClick={handleSubmit(updateLFParameters, onValidationError)}
+                                    variant="outlined"
+                                    disabled={loadFlowStatus === RunningStatus.RUNNING}
+                                >
+                                    <FormattedMessage id="validate" />
+                                </SubmitButton>
+                            </Grid>
+                        </Box>
+                    </Box>
+                </Grid>
+                {openCreateParameterDialog && (
+                    <CreateParameterDialog
+                        open={openCreateParameterDialog}
+                        onClose={() => setOpenCreateParameterDialog(false)}
+                        parameterValues={() => {
+                            return { ...params };
+                        }}
+                        parameterFormatter={(newParams) => newParams}
+                        parameterType={ElementType.LOADFLOW_PARAMETERS}
+                    />
+                )}
+
+                {openSelectParameterDialog && (
+                    <DirectoryItemSelector
+                        open={openSelectParameterDialog}
+                        onClose={handleLoadParameter}
+                        types={[ElementType.LOADFLOW_PARAMETERS]}
+                        title={intl.formatMessage({
+                            id: 'showSelectParameterDialog',
+                        })}
+                        onlyLeaves={true}
+                        multiselect={false}
+                        validationButtonText={intl.formatMessage({
+                            id: 'validate',
+                        })}
+                    />
+                )}
+            </CustomFormProvider>
+        </LoadFlowProvider>
+    );
```

Comment:
the JSX is very big here.
Maybe we could have several rendering methods: renderProvider, renderTabs, ... render Buttons ?